### DISCUSSION TITLE
is the c# .net wrapper maintained and updated with new features and versions?

### DISCUSSION THREAD

**Discussion by agonzalezm** [explosion/spaCy#11657] [Help: Other Questions] [answered]
I am testing the c# spacy wrapper and looks good but before using it in a production solution would like to know if it is going to be maintained in future and updated or is there any better solution for c#?

**Comment by polm** [comment, marked as answer]
I guess you're referring to [this project](https://spacy.io/universe/project/SpacyDotNet)? That's a third party project and not maintained by the spaCy team directly, so we can't make any promises about maintenance. I'm not aware of any other C# wrapper for spaCy.
